docs/blocks/block-development/development.md

```diff
@@ -14,7 +14,7 @@ Any required Python dependencies for your block can be added to your `requiremen
 
 Other elements of a block to keep in mind include [block patterns](block-patterns.md), the [nio framework](/blocks/block-development/framework.md) (including [discoverability](/blocks/block-development/framework.md#discoverability), and [mixins](/blocks/block-development/mixins.md).
 
-Once you have developed your block, you will want to [test](/blocks/block-development/block-testing.md) and [document](/blocks/block-development/documenting.md) your block.
+Once you have developed your block, you will want to [test](/blocks/block-development/block-testing.md) and [document](/blocks/block-development/documenting.md) your block. Blocks can be manually installed for local use in the nio System Designer by adding them as a submodule with the `git submodule add [repository link]` command. Once you have completed, tested, and documented your block, E-mail the github respository link to blocks@n.io for approval of submission to the nio Block Library.
```

Comment:
'adding them as a submodule'—where do you run this command? does the cli `nio add` from the project root work just as well? The goal being to favor the cli over git whenever possible.

Reply:
nio add only adds blocks from the nio-blocks repository.